lib/hiera/config.rb

```diff
@@ -74,7 +74,7 @@ def load_backends
         begin
           require "hiera/backend/#{backend.downcase}_backend"
         rescue LoadError => e
-          Hiera.warn "Cannot load backend #{backend}: #{e}"
+          fail "Cannot load backend #{backend}: #{e}"
```

Comment:
Was using `fail` vs `raise` here a conscious decision? Fwiw I don't see any other uses of `fail` in hiera.